tailwind.config.js

```diff
@@ -1,5 +1,5 @@
 const BASE_FONT_SIZE = 16;
-const pxToRem = (px) => `${ px / BASE_FONT_SIZE }rem`;
+const pxToRem = (px) => `${px / BASE_FONT_SIZE}rem`;
```

Comment:
Why did you remove the spaces in the template literal? Please put them back

Reply:
Sorry, I used prettier